src/main/resources/db/changelog/db.changelog-master.yaml

```diff
@@ -1,4 +1,11 @@
 databaseChangeLog:
   - include:
       file: db/changelog/changes/01-create-books-table.yaml
+  - include:
+      file: db/changelog/changes/02-create-user-table.yml
+  - include:
+      file: db/changelog/changes/03-create-roles-table.yml
+  - include:
+      file: db/changelog/changes/04-create-users-roles-table.yml
+

```

Comment:
consider adding `insert user, role, books` scripts

Reply:
Do you mean to insert test data for   user, role, books scripts?